File: experiments/020-PythonVsJava/cells/java-gradle_1-Empty_free/2019-2c-parcial2-masked/20260921-212555-19812/workdir/src/main/java/surveycrawler/Boulder.java
package surveycrawler;

public class Boulder extends Ground {

    // moving

    @Override
    public void moveCrawlerTowards(SurveyCrawler aCrawler, Point aDirection) {
        aCrawler.signalBoulderFound();
    }

    // facing

    @Override
    public void turnCrawlerClockwise(SurveyCrawler aCrawler) {
        aCrawler.signalBoulderFound();
    }

    @Override
    public void turnCrawlerCounterClockwise(SurveyCrawler aCrawler) {
        aCrawler.signalBoulderFound();
    }
}

File: experiments/020-PythonVsJava/cells/python-pytest_1-Empty_free/2019-2c-parcial2-masked/20260921-204513-77784/analysis/acceptance-project/src/surveycrawler/boulder.py
from surveycrawler.ground import Ground


class Boulder(Ground):

    # moving

    def move_crawler_towards(self, a_crawler, a_direction):
        a_crawler.signal_boulder_found()

    # facing

    def assert_crawler_can_turn(self, a_crawler):
        a_crawler.signal_boulder_found()
